lib/src/chart/line_chart/line_chart_painter.dart

```diff
@@ -200,17 +215,22 @@ class LineChartPainter extends AxisChartPainter<LineChartData> with TouchHandler
   /// first one is the sharp corners line on spot connections
   /// second one is curved corners line on spot connections,
   /// and we use isCurved to find out how we should generate it,
-  Path _generateBarPath(Size viewSize, LineChartBarData barData) {
+  /// If you want to concatenate paths together for creating an area between
+  /// multiple bars for example, you can pass the appendToPath
+  Path _generateBarPath(Size viewSize, LineChartBarData barData, Path appendToPath) {
```

Comment:
May I ask you put the `appendToPath` inside { }, to make it optional? (default null)